spec/Occupant/Identifier/Identifier.vspec

```diff
@@ -0,0 +1,19 @@
+#
+# (C) 2020 Robert Bosch GmbH
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+#
+# Identifier
+#
+
+- Subject:
+  datatype: string
+  type: sensor
+  description: Subject for the authentification of the occupant.
+  
```

Comment:
@gunnarx: improve the documentation (point to OAuth)

Reply:
@danielwilms added OAuth 2.0